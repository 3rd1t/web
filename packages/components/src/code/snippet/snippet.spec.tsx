import React from "react"
import Snippet from "./snippet"
import { render } from "@testing-library/react"

describe(" Snippet", () => {
  it("renders correctly", () => {
    const { baseElement } = render(<Snippet language="js">"const x = 1;"</Snippet>)
    expect(baseElement).toBeTruthy()
  })
})
