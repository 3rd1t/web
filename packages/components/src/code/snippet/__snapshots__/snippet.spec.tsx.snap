// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[` Snippet renders correctly 1`] = `
<body>
  <div>
    <pre
      class="font-mono"
      data-testid="syntaxHighlighter"
      style="font-size: 14px; line-height: 1.375; direction: ltr; text-align: left; white-space: pre; word-spacing: normal; word-break: normal; tab-size: 4; hyphens: none; color: rgb(203, 213, 224); padding: 1em; margin: .5em 0px; overflow: auto;"
    >
      <code
        class="language-text"
        style="font-size: 14px; line-height: 1.375; direction: ltr; text-align: left; white-space: pre; word-spacing: normal; word-break: normal; tab-size: 4; hyphens: none; color: rgb(203, 213, 224);"
      >
        "const x = 1;"
      </code>
    </pre>
  </div>
</body>
`;
